modules/emulator/R/calcSpatialCov.list.R

```diff
@@ -11,25 +11,18 @@
 ##' @param tau spatial var
 ##' 
 ##' @author Michael Dietze
-`calcSpatialCov.list` <-
-function(d,psi,tau){
-  m <- length(d)
+calcSpatialCov.list <- function(d, psi, tau) {
+  m  <- length(d)
   nl <- nrow(d[[1]])
-  H <- matrix(0,nl,nl)
-  if(length(psi) == 1 && m > 1){
-    ##apply same psi to all directions
-    psi <- rep(psi,m)
+  H  <- matrix(0, nl, nl)
+  if (length(psi) == 1 && m > 1) {
+    ## apply same psi to all directions
+    psi <- rep(psi, m)
   }
-  #for(i in 1:nl){
-  #  for(j in i:nl){
-  #    tmp <- 0
-  #    for(k in 1:m){tmp <- tmp - psi[k]*d[[k]][i,j]}
-  #    H[i,j] <- tau*exp(tmp)
-  #    H[j,i] <- H[i,j]
-  #  }
-  #}
-  for(k in 1:m) H <- H - psi[k]*d[[k]]
-  H <- tau*exp(H)
-  return(H)
-}
-
+  # for(i in 1:nl){ for(j in i:nl){ tmp <- 0 for(k in 1:m){tmp <- tmp -
```

Comment:
:+1: to leaving commented code blocks formatted as they were originally.

I think a good workflow with `formatR` is to run it and then stage code interactively (`git add --patch`) before committing. That way, reasonable changes can be committed and weird artifacts like this can be easily undone after the commit with `git checkout --` or `git reset --hard`.